### PR TITLE
Fixed the DNSHits issue in UWC

### PR DESCRIPTION
**Pull request by nagdeepgk**
Removed the unused interfaces from InfluxDBConnector container.  This is a replica of the PR https://github.com/open-edge-insights/uwc/pull/23. However, this is based out of develop branch instead of master. 

 Signed-off-by: nagdeepgk <nagdeep.gk@intel.com>

### CODE REVIEW

File: docker-compose_UT.yml [hunk 1]
```diff
@@ -178,7 +184,7 @@ services:
     environment:
       AppName: "MQTT"
       DEV_MODE: ${DEV_MODE}
-      no_proxy: ${eii_no_proxy}
+      no_proxy: ${eii_no_proxy},${ETCD_HOST}
```
Comment (vkb1) [inline docker-compose_UT.yml:187]: I see above that you have removed ${eii_no_proxy} from the list but see here you have retained, we can be consistent

File: docker-compose_UT.yml [hunk 2]
```diff
@@ -79,31 +79,33 @@ services:
...
+      no_proxy: ${ETCD_HOST}
       ETCD_HOST: ${ETCD_HOST}
       ETCD_CLIENT_PORT: ${ETCD_CLIENT_PORT}
       ETCD_PREFIX: ${ETCD_PREFIX}
+      ETCD_ENDPOINT: "$ETCD_HOST:$ETCD_CLIENT_PORT"
     networks:
       - eii
     volumes:
-    - "vol_etcd_keeper:/tmp/nginx"
-    user: ${EII_UID}
```
Comment (vkb1) [inline docker-compose_UT.yml:106]: Are you removing the user id purposefully?

File: docker-compose_UT.yml [hunk 3]
```diff
@@ -115,7 +117,7 @@ services:
     - etcd_etcduser_cert
     - etcd_etcduser_key
 
-  ia_eii_zmq_broker:
+  ia_zmq_broker:
```
Comment (vkb1) [inline docker-compose_UT.yml:120]: Hope you have done sanity to ensure that this name change doesn't impact anything w.r.t functionality

File: docker-compose_UT.yml [hunk 4]
```diff
@@ -125,29 +127,33 @@ services:
...
+      no_proxy: ${ETCD_HOST}
       ETCD_HOST: ${ETCD_HOST}
       ETCD_CLIENT_PORT: ${ETCD_CLIENT_PORT}
       ETCD_PREFIX: ${ETCD_PREFIX}
+    ports:
+      - 60514:60514
+      - 60515:60515
     networks:
-      - eii
+    - eii
```
Comment (vkb1) [inline docker-compose_UT.yml:156]: Let's undo this

File: docker-compose_UT.yml [hunk 5]
```diff
@@ -287,7 +295,7 @@ services:
     image: ${DOCKER_REGISTRY}modbus-tcp-master-test:${EII_VERSION}
     container_name: modbus-tcp-master-test
     hostname: modbus-tcp-master-test
-    # read_only: true
+    #read_only: true
```
[the remark this answers is not in the document]
Comment (nagdeepgk) [reply docker-compose_UT.yml:298]: done

File: docker-compose_UT.yml [hunk 6]
```diff
@@ -79,31 +79,33 @@ services:
...
+    image: ${DOCKER_REGISTRY}openedgeinsights/ia_etcd_ui:${EII_VERSION}
     container_name: ia_etcd_ui
     hostname: ia_etcd_ui
     restart: unless-stopped
     security_opt:
     - no-new-privileges
     healthcheck:
       test: ["CMD-SHELL", "exit", "0"]
-      interval: 5m
+      interval: 5m
```
[the remark this answers is not in the document]
Comment (nagdeepgk) [reply docker-compose_UT.yml:94]: done

File: docker-compose_UT.yml [hunk 7]
```diff
@@ -79,31 +79,33 @@ services:
         ETCD_VERSION: ${ETCD_VERSION}
         EII_VERSION: ${EII_VERSION}
         EII_UID: ${EII_UID}
-        DOCKER_REGISTRY: ${DOCKER_REGISTRY}
```
[the remark this answers is not in the document]
Comment (nagdeepgk) [reply docker-compose_UT.yml:82]: actually all changes to this file docker-compose_UT.yml have been done and released in1.6 itself. We are seeing this file now again in this PR bcoz, this commit was added directly added to master (on tag day caught during sanity from us), hence instead of adding to develop branch again and re- rebasing to master we added directly into develop branch.  But now as part of this process to keep develop not missing any commits from master I did cherry-pick this commit hence we can see this here.
in short this is already released file in 1.6 PV.

File: docker-compose_UT.yml (hunk 2 above)
Comment (nagdeepgk) [reply docker-compose_UT.yml:106]: yes, we tried to adhere to how these vol mounts look in original docker-compose.yml of etcd-UI - https://github.com/open-edge-insights/eii-zmq-broker/blob/master/docker-compose.yml.
Since this is not having eii_uid, we removed form here too.

**Comment by nagdeepgk** [comment]
@vkb1 : a general note. This PR ideally should have only the influcDBconn.json but docker-compose_UT.yml is seen bcoz,
UT file was already part of the 1.6 tag. Tag commit: https://github.com/open-edge-insights/uwc/blob/master/docker-compose_UT.yml.
However, since this commit was only in master & not in develop, i cherry-picked it to make master and develop branches have same commits.

**Review by aditisa1 — APPROVED** [review]

**Comment by nagdeepgk** [comment]
closing this as this is already merged in naother pr - https://github.com/open-edge-insights/uwc/commit/eed6a5eecd49b24c08b807753be745799db925e7 & https://github.com/open-edge-insights/uwc/commit/5e586cfb5300de997674b50e8248c1dd76fac765
